lib/sandbox.js

```diff
@@ -1280,8 +1280,6 @@ function sandBox(script, name, verbose, debug, context) {
                 id = adapter.namespace + '.' + id;
             }
 
-            if (clearRunning === undefined) clearRunning = true;
```

Comment:
This will never happen because `typeof clearRunning !== boolean` is already checked above.